### DISCUSSION TITLE
False memory alert on Data Engine

### DISCUSSION THREAD

**Discussion by agauttam** [utmstack/UTMStack#701] [General]
Hi,

I am getting an alert on the UTMStack GUI for memory, but when I checked the utilization on the container, it is normal.

![image](https://github.com/utmstack/UTMStack/assets/168720506/e28755d4-8d31-44cd-8f7f-b719c7fae74b)

![memory_alert](https://github.com/utmstack/UTMStack/assets/168720506/e51cc6f9-01d8-4fe7-9a9b-201baf4dbaaf)

**Comment by c3s4rfred** [comment]
This is a normal behavior @agauttam, the Data Engine can have memory pikes during data processing, and the refreshing rate is not the same between the application and the operating system, but we will check anyway, thanks for the feedback.

**Reply by agauttam** [reply]
Thanks for the update Freddy.

Seems its an issue of fetching parameter. I have same experience when we collect the details through SNMP in Linux for CPU and memory utilization .
